RileyLinkKit/PumpOpsSynchronous.swift

```diff
@@ -501,9 +502,12 @@ class PumpOpsSynchronous {
                     }
                 }
 
-                if let event = event as? ChangeTimePumpEvent {
-                    timeAdjustmentInterval += event.adjustmentInterval
+                if let event = event as? NewTimePumpEvent {
+                    if let ct_event = lastEvent as? ChangeTimePumpEvent {
```

Comment:
You don't need two separate if statements here, you can have two lets separated by a comma.